_test/test_combine/test_combine_sqw.m

```diff
@@ -63,17 +63,32 @@
             
         end
         %
-        function DISABLED_test_combine1D(obj)
+        function test_combine1D(obj)
             % this is to fix in a future. There is problem with orhtogonal
```

Comment:
Is this now fixed? 
"orthogonal"
A future "patch"?

Reply:
spurious comment left. Modified and fixed